hdf/test/testhdf.c

```diff
@@ -268,8 +268,12 @@ main(int argc, char *argv[])
 
     if (CleanUp) {
         MESSAGE(2, printf("\nCleaning Up...\n\n"););
+#if !(defined DOS386 || defined WIN386)
+        system("rm -f *.hdf *.tmp");
+#else
         remove("*.hdf");
         remove("*.tmp");
+#endif
```

Comment:
We know the else clause doesn't work, so i'd just delete it. On Windows, files just won't be cleaned up until we really fix file cleanup.